no-computer.md

```diff
@@ -1,43 +1,51 @@
 # No computer project
-
+Math problem sets as a DSL
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The need is for communication between teachers and students about what work needs to be done in order to succeed in the class. Problem sets help both teachers and students in quantitative courses by providing them with means to enumerate what tasks need to be done for homework and how they should be done. Without problem sets math students would be left trying to find good problems to help them learn the material while professors would be forced to deal with students that were all doing different work. With problem sets it helps professors and graders be of more help to the students as well because they can be very familiar with the problems that students are doing and therefore be able to provide tips without giving away the answer or disallowing the students from learning.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A language meets their needs because it allows the professor to tell all of his students exactly what to do, have them understand exactly what needs to be done, and do it in a specific way that is understandable to the professor, grader, and other students.
```

Comment:
I don't think this is a language because every execution of the same "script" should yield the same result hopefully. If the language itself doesn't involve the student solving the pset, then the execution of the language is so random, that it is pointless in even calling this a language. What is the grader part then?